### PR TITLE
[DOCS] Fix obsolete Slack setup info

### PR DESCRIPTION
**Pull request by ppf2**
Slack no longer recommends the legacy "integrations" setup (https://api.slack.com/legacy/custom-integrations/incoming-webhooks).  Updated documentation to reference https://api.slack.com/messaging/webhooks instead.  

Removed screenshots from our documentation related to Slack setup.  We should avoid these screenshots (and simply point to Slack documentation) for Slack may change the instructions/their UI in the future.

Also, added a short note on the use case of notifying multiple Slack channels.  

I think we will want to backport this doc change given the following statement in Slack documentation:

image

## Preview

https://elasticsearch_57834.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/actions-slack.html

### CODE REVIEW

File: x-pack/docs/en/watcher/actions/slack.asciidoc [hunk 1]
```diff
@@ -181,20 +181,8 @@ aggregation and the Slack action:
...
-. Go to https://my.slack.com/services/new/incoming-webhook.
-. Select a default channel for the integration.
-+
-image::images/slack-add-webhook-integration.jpg[]
-. Click *Add Incoming Webhook Integration*.
-. Copy the generated webhook URL so you can paste it into your Slack account
-  configuration in `elasticsearch.yml`.
-+
-image::images/slack-copy-webhook-url.jpg[]
+You need a Slack App with the Incoming Webhooks feature[https://api.slack.com/messaging/webhooks]
```
Comment (lcawl) [inline x-pack/docs/en/watcher/actions/slack.asciidoc:184]:
```suggestion
You need a Slack App with the https://api.slack.com/messaging/webhooks[Incoming Webhooks feature] 
```

**Review by dakrone — APPROVED** [review]
LGTM (with Lisa's fix), thanks Pius

File: x-pack/docs/en/watcher/actions/slack.asciidoc [hunk 2]
```diff
@@ -181,20 +181,8 @@ aggregation and the Slack action:
...
-. Select a default channel for the integration.
-+
-image::images/slack-add-webhook-integration.jpg[]
-. Click *Add Incoming Webhook Integration*.
-. Copy the generated webhook URL so you can paste it into your Slack account
-  configuration in `elasticsearch.yml`.
-+
-image::images/slack-copy-webhook-url.jpg[]
+You need a Slack App with the Incoming Webhooks feature[https://api.slack.com/messaging/webhooks] 
+to configure a Slack account. Use the generated webhook URL to set up your Slack account in Elasticsearch.
```
Comment (jrodewig) [inline x-pack/docs/en/watcher/actions/slack.asciidoc]:
```suggestion
You need a Slack App with the Incoming Webhooks feature[https://api.slack.com/messaging/webhooks] 
to configure a Slack account. Use the generated webhook URL to set up your Slack account in {es}.
```

File: x-pack/docs/en/watcher/actions/slack.asciidoc [hunk 3]
```diff
@@ -231,6 +219,9 @@ xpack.notification.slack:
           mrkdwn_in: "pretext, text"
 --------------------------------------------------
 
+To notify multiple channels, create a webhook URL for each channel in Slack 
+and multiple Slack accounts in Elasticsearch (one for each webhook URL).
```
Comment (jrodewig) [inline x-pack/docs/en/watcher/actions/slack.asciidoc:223]:
```suggestion
To notify multiple channels, create a webhook URL for each channel in Slack 
and multiple Slack accounts in {es} (one for each webhook URL).
```

File: x-pack/docs/en/watcher/actions/slack.asciidoc [hunk 4]
```diff
@@ -181,20 +181,8 @@ aggregation and the Slack action:
...
-. Select a default channel for the integration.
-+
-image::images/slack-add-webhook-integration.jpg[]
-. Click *Add Incoming Webhook Integration*.
-. Copy the generated webhook URL so you can paste it into your Slack account
-  configuration in `elasticsearch.yml`.
-+
-image::images/slack-copy-webhook-url.jpg[]
+You need a Slack App with the https://api.slack.com/messaging/webhooks[Incoming Webhooks feature] 
+to configure a Slack account. Use the generated webhook URL to set up your Slack account in Elasticsearch.
```
Comment (jrodewig) [inline x-pack/docs/en/watcher/actions/slack.asciidoc:185]:
```suggestion
You need a Slack App with the https://api.slack.com/messaging/webhooks[Incoming Webhooks feature] 
to configure a Slack account. Use the generated webhook URL to set up your Slack account in {es}.
```

**Comment by jrodewig** [comment]
@elasticmachine update branch

**Review by jrodewig — APPROVED** [review]
LGTM. I'll merge once I get the CI to pass. Thanks @ppf2!

**Pull request merged**